tests_config/run_spec.js

```diff
@@ -10,12 +10,15 @@ const AST_COMPARE = process.env["AST_COMPARE"];
 const VERIFY_ALL_PARSERS = process.env["VERIFY_ALL_PARSERS"] || false;
 const ALL_PARSERS = process.env["ALL_PARSERS"]
   ? JSON.parse(process.env["ALL_PARSERS"])
-  : ["flow", "babylon", "typescript"];
+  : ["flow", "graphql", "babylon", "typescript"];
 
 function run_spec(dirname, options, additionalParsers) {
   fs.readdirSync(dirname).forEach(filename => {
     const extension = extname(filename);
-    if (/^\.([jt]sx?|css)$/.test(extension) && filename !== "jsfmt.spec.js") {
+    if (
+      /^\.([jt]sx?|css|graphql)$/.test(extension) &&
```

Comment:
looks like we need to add .less, .scss in that list as well (not related to this PR)